src/sentry/snuba/metrics/query.py

```diff
@@ -243,7 +243,7 @@ def validate_orderby(self) -> None:
                 self._validate_field(metric_order_by_field.field)
 
         orderby_metric_fields: Set[MetricField] = set()
-        metric_entities: Set[MetricField] = set()
+        metric_entities: Set[MetricEntity] = set()
```

Comment:
@obostjancic @iambriccardo Could you please check this type change?

Reply:
👍 Good catch!